### PR TITLE
Fix UG bugs and update my PPP

### LINKED ISSUE #148: [PE-D][Tester E] pending function not implemented or not seen despite being in the UG
Not sure if I missed it but I do not see any indicators on the GUI for the pending status. 

But it seems like the feature is either not implemented or documentation is not updated to show user how it works

-------------
Labels: `type.FeatureFlaw` `severity.High`
original: WesleyBLDC/ped#7

### LINKED ISSUE #158: [PE-D][Tester E] Ug lacks examples
Confusing to user, there should be examples given instead of only format.

Recommend to add atleast 1 or 2 examples to each command in the userguide

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: WesleyBLDC/ped#11

### LINKED ISSUE #165: [PE-D][Tester C] mark command discrepancy in the UG
UG specifies the mark command format as `mark INDEX`, when it is `mark m/INDEX` in implementation.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: hansstanley/ped#11

### LINKED ISSUE #171: [PE-D][Tester B] pending command not working as stated in the UG nor is it working as stated in the application itself
the UG states that pending command shows meetings that either have dates that are in the past or not marked as done. However, when adding a meeting that has a date that is in the past, when i typed the command "pending", this particular meeting is not shown.

However, the results tab shows that the pending command shows "all un-done or not over meetings". This particular meeting is in the past, but not marked as done, hence it should still show up as stated in the results tab

Replication 1 based on UG:
1. addm m/CS2103T Tutorial p/John Doe p/Mary Jane dt/22-02-2022 22:22 l/COM1-B103 des/CS2101 Tutorial. <- adds a meeting with a past date
3. pending

Expectation 1 based on UG:
Since UG states that  it shows meetings that have dates in the past or not done, which in this case, the meeting satisfies both conditions, it should be shown

Actual 1 based on UG:
This meeting is not shown

Replication 2 based on results display definition (Showing all un-done or not over meetings):
1.  m/CS2103T Tutorial p/John Doe p/Mary Jane dt/22-02-2024 22:22 l/COM1-B103 des/CS2101 Tutorial. <- adds a meeting with a future date that hasnt passed yet
2. mark m/1
3. pending

Expectation 2 based on results display:
This particular meeting satisfies the second condition of the results display definition (not over meetings) as the date has not passed yet. Hence it should be displayed

Actual 2:
Not displayed

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: chongweiguan/ped#10

### LINKED ISSUE #172: [PE-D][Tester B] Command summary missing mark, unmark and findm commands
As per the title, command summary missing mark, unmark and findm commands

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: chongweiguan/ped#11

### PR DESCRIPTION
**Pull request by chuakid**
Fixes the following issues:
- [x] Closes #165
- [x] Closes #148 
- [x] Closes #172  
- [x] Closes #158 
- [x] Closes #171

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -647,28 +677,31 @@ Format: `exit`
...
+| **Edit a meeting**           | `editm INDEX [m/TITLE] [dt/DATE_TIME] [p/CONTACT_NAME]... [l/LOCATION] [des/DESCRIPTION]`                                                                             |
+| **Find a meeting**           | `findm KEYWORD [MORE_KEYWORDS]` <br> e.g, `findm James Jake`                                                                                                          |
+| **List all meetings**        | `findm`                                                                                                                                                               |
+| **Mark meeting as done**     | `mark m/INDEX`                                                                                  <br/>                                                                 |
+| **Mark meeting as not done** | `unmark m/INDEX`                                                                    <br/>                                                                             |
```
Comment (Beebeeoii) [inline docs/UserGuide.md:701]: Mark/unmark can be done for multiple meetings at one go.

should be `mark m/INDEX [m/MORE_INDEXES]...` and `unmark m/INDEX [m/MORE_INDEXES]...`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -590,18 +617,19 @@ Examples:
...
+Format: "mark m/INDEX..." or "unmark m/INDEX..."
 
 * The index refers to the index number shown in the displayed meetings list.
 * The index must be a positive integer 1, 2, 3...
 
-### Showing Meetings that are done or over : `pending`
+### Showing pending meetings : `pending`
 
-Shows meetings that either have dates that are in the past or not marked as done.
+Shows meetings that are not marked as done and are in the future i.e not over yet.
```
Comment (Beebeeoii) [inline docs/UserGuide.md:629]:
```suggestion
Shows meetings that are not marked as done and those that are in the future i.e not over yet.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -590,18 +617,19 @@ Examples:
...
-Format: "mark INDEX" or "unmark INDEX"
+Marks the specified meeting as either done or undone. Meetings marked as done will have green colored titles.
+![img.png](images/markasdone.png)
+Format: "mark m/INDEX..." or "unmark m/INDEX..."
 
 * The index refers to the index number shown in the displayed meetings list.
 * The index must be a positive integer 1, 2, 3...
 
-### Showing Meetings that are done or over : `pending`
+### Showing pending meetings : `pending`
```
Comment (Beebeeoii) [inline docs/UserGuide.md:627]: can we put a `TIP` box to tell user how to get back to the default view of all the meetings? aka tell them to use `findm` to list all

File: docs/UserGuide.md [hunk 4]
```diff
@@ -590,18 +617,19 @@ Examples:
 
 ### Marking Meetings as done or undone : `mark` or `unmark`
 
-Marks the specified meeting as either done or undone.
-
-Format: "mark INDEX" or "unmark INDEX"
+Marks the specified meeting as either done or undone. Meetings marked as done will have green colored titles.
+![img.png](images/markasdone.png)
+Format: "mark m/INDEX..." or "unmark m/INDEX..."
```
Comment (Beebeeoii) [inline docs/UserGuide.md:622]:
```suggestion
Format: "mark m/INDEX [m/MORE_INDEXES]..." or "unmark m/INDEX [m/MORE_INDEXES]..."
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -590,18 +617,19 @@ Examples:
 
 ### Marking Meetings as done or undone : `mark` or `unmark`
 
-Marks the specified meeting as either done or undone.
-
-Format: "mark INDEX" or "unmark INDEX"
+Marks the specified meeting as either done or undone. Meetings marked as done will have green colored titles.
```
Comment (Beebeeoii) [inline docs/UserGuide.md:620]: can we put a `NOTE` box to emphasis that marked meetings have green titles?

**Review by Beebeeoii — CHANGES_REQUESTED** [review]
lgtm other than a few nits. thanks !

File: docs/UserGuide.md (hunk 5 above)
Comment (chuakid) [reply docs/UserGuide.md]: Resolved 7723b1005a3a3e06d20074b506c386d038715118

File: docs/UserGuide.md (hunk 3 above)
Comment (chuakid) [reply docs/UserGuide.md:632]: Resolved 4a508f1

File: docs/UserGuide.md (hunk 1 above)
Comment (chuakid) [reply docs/UserGuide.md]: Fixed 602e3d6

**Pull request merged**